src/main/java/seedu/address/logic/parser/ArgumentMultimap.java

```diff
@@ -57,4 +61,18 @@ public List<String> getAllValues(Prefix prefix) {
     public String getPreamble() {
         return getValue(new Prefix("")).orElse("");
     }
+
+    /**
+     * Throws a {@code ParseException} if any of the prefixes given in {@code prefixes} appeared more than
+     * once among the arguments.
+     */
+    public void verifyNoDuplicatePrefixesFor(Prefix... prefixes) throws ParseException {
```

Comment:
Could consider shortening the name of the method to verifyNoDuplicatePrefixes as the 'For' does not seem to be very necessary

Reply:
It reads as a natural statement e.g., `verifyNoDuplicatePrefixesFor(PREFIX_NAME, PREFIX_ADDRESS)` i.e., verify no duplicate prefixes for A, B, ..
But OK to remove. 
Can also consider `verifyNoDuplicatesFor` as Prefix is implied by the parameter type.